.goreleaser.yml

```diff
@@ -0,0 +1,36 @@
+project_name: management-interface
+
+release:
+  github:
+    owner: TheCacophonyProject
+    name: management-interface
+  name_template: '{{.Tag}}'
+
+builds:
+- goos:
+  - linux
+  goarch:
+  - arm
+  goarm:
+  - "7"
+  main: cmd/managementd/main.go
+  ldflags: -s -w -X main.version={{.Version}}
+  binary: management-interface
+
+nfpm:
+  vendor: The Cacophony Project
+  homepage: http://cacophony.org.nz/
+  maintainer: Cameron Ryan-Pears <cameron@cacophony.org.nz>
```

Comment:
We should probably use "Cacophony Developers <dev@cacophony.org.nz>" for things like this.
I've started doing this for recent deployment/release stuff.